### PR TITLE
Fix FluenceJS changelog

### PR DESCRIPTION
**Pull request by coder11**

### CODE REVIEW

File: docs/build/fluence-js/changelog.md [hunk 1]
```diff
@@ -2,27 +2,193 @@
...
 
-* `0` shows that Fluence JS does not meet its vision yet, so API can change quickly
-* `BREAKING` part is incremented for each breaking API change
-* `ENHANCING` part is incremented for every fix and update which is compatible on API level
+- `0` shows that Fluence JS does not meet its vision yet, so API can change quickly
+- `BREAKING` part is incremented for each breaking API change
+- `ENHANCING` part is incremented for every fix and update which is compatible on API level
 
-## [0.20.2](https://github.com/fluencelabs/fluence-js/releases/tag/v0.19.1) – February 23, 2022
+## [0.26.3](https://github.com/fluencelabs/fluence-js/releases/tag/0.26.3) - Oct 26, 2022
```
Comment (shamsartem) [inline docs/build/fluence-js/changelog.md]: These links are unreachable:

https://github.com/fluencelabs/fluence-js/releases/tag/0.26.3
https://github.com/fluencelabs/fluence-js/releases/tag/0.24.0
https://github.com/fluencelabs/fluence-js/releases/tag/0.24.1
https://github.com/fluencelabs/fluence-js/releases/tag/0.26.2
https://github.com/fluencelabs/fluence-js/releases/tag/0.26.0
https://github.com/fluencelabs/fluence-js/releases/tag/0.25.2
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.7
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.8
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.2
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.6
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.4
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.5
https://github.com/fluencelabs/fluence-js/releases/tag/0.21.7
https://github.com/fluencelabs/fluence-js/releases/tag/0.22.0
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.0
https://github.com/fluencelabs/fluence-js/releases/tag/0.21.8
https://github.com/fluencelabs/fluence-js/releases/tag/0.25.1
https://github.com/fluencelabs/fluence-js/releases/tag/0.26.1
https://github.com/fluencelabs/fluence-js/releases/tag/0.23.1
https://github.com/fluencelabs/fluence-js/releases/tag/0.21.4
https://github.com/fluencelabs/fluence-js/releases/tag/0.21.6

File: docs/build/fluence-js/changelog.md [hunk 2]
```diff
@@ -2,27 +2,193 @@
...
+
+## [0.26.2](https://github.com/fluencelabs/fluence-js/releases/tag/0.26.2) - Oct 9, 2022
+
+Update AVM to 0.30.11 ([#191](https://github.com/fluencelabs/fluence-js/pull/191))
+
+## [0.26.1](https://github.com/fluencelabs/fluence-js/releases/tag/0.26.1) - Oct 9, 2022
+
+**Compiler support:**
+
+Add v4 version of compiler support. It makes possible to use any name for functions and data structures in aqua without colliding with variable names in FluenceJS ([#190](https://github.com/fluencelabs/fluence-js/pull/190))
```
Comment (shamsartem) [inline docs/build/fluence-js/changelog.md]: Add v4 of compiler support. It makes it possible to use any name for functions and data structures in aqua without collisions with variable names in FluenceJS

**Review by shamsartem — CHANGES_REQUESTED** [review]
Please fix or remove the links

File: docs/build/fluence-js/changelog.md (hunk 1 above)
Comment (folex) [reply docs/build/fluence-js/changelog.md]: They all miss `v` before version

**Review by shamsartem — APPROVED** [review]
Cool thanks

**Review by folex — APPROVED** [review]

**Pull request merged**